test/integration/localnet/get-version.test.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * LocalNet Integration Test for GetVersion API
+ *
+ * This test validates connectivity to a running LocalNet instance by calling the getVersion API and validating the full
+ * response structure.
+ *
+ * Prerequisites:
+ *
+ * - LocalNet must be running (set up by CircleCI or manually via scripts)
+ * - Environment variables must be configured (.env.localnet in CI)
+ *
+ * This test verifies:
+ *
+ * - SDK can connect to LocalNet JSON API
+ * - GetVersion API returns a complete and valid response
+ * - Response contains expected version string format and features structure
+ */
+
+import { LedgerJsonApiClient } from '../../../src/clients/ledger-json-api';
+import { EnvLoader } from '../../../src/core/config/EnvLoader';
+
+describe('LocalNet GetVersion Integration Test', () => {
+  let client: LedgerJsonApiClient;
+
+  beforeAll(() => {
+    // Configure for LocalNet environment
+    process.env['CANTON_CURRENT_NETWORK'] = 'localnet';
+    process.env['CANTON_CURRENT_PROVIDER'] = 'app-provider';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_LEDGER_JSON_API_URI'] = 'http://localhost:39750';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_LEDGER_JSON_API_CLIENT_ID'] = 'admin';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_LEDGER_JSON_API_CLIENT_SECRET'] = 'admin';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_AUTH_URL'] = 'http://localhost:3000/auth';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_PARTY_ID'] = 'test::party::123';
+    process.env['CANTON_LOCALNET_APP_PROVIDER_USER_ID'] = 'test-user';
+
+    const config = EnvLoader.getConfig('LEDGER_JSON_API');
+    client = new LedgerJsonApiClient(config);
+  });
+
+  it('should successfully call getVersion and validate full response structure', async () => {
+    // Call getVersion API
+    const response = await client.getVersion();
+
+    // Validate response is defined
```

Comment:
Simplify all these checks into a single check

expect(response).to.eq({
  version: "3.0.3",
  ...